latex/Research_Strategy.tex

```diff
@@ -257,14 +265,16 @@
 %References
 Additionally, we try to draw attention to and promote analysis of model uncertainty (as distinct from statistical uncertainty) in calculated values [refs], as understanding the confidence levels of predictions is particularly important for guiding molecular design.
 %Cite model uncertainty stuff.
+{\color{red}[JDC: Tracking down \emph{why} models are failing and what particular problems still need attention has also been an extremely powerful aspect of SAMPL. Much of this is left to the participants, of course, but we should mention the work the organizers do in spotting patterns and providing a venue for the participants to explore these issues.]}
 
 
 {\large \bf APPROACH}
 % 9 pages
 
-Our approach to systematically advancing modeling for biomolecular design involves collecting carefully tailored experimental data for challenges focusing on physical property prediction, host-guest binding, and biomolecular binding.
-Thus, we have four main aims, three of which focus on tailoring and generating this experimental data, and a fourth which focuses on running SAMPL challenges to advance modeling and maximize the learning value to the community.
-These SAMPL challenges will run yearly (though not necessarily every aspect of the proposal will feature in every SAMPL challenge), so each of the aims focused on generating experimental data has a series of stages which correspond to individual future SAMPL challenges.
+Our approach to systematically advancing modeling for biomolecular design involves collecting carefully targeted experimental datasets for challenges focusing on physical property prediction, host-guest binding, and protein-ligand binding that isolate individual limitations in quantitative physical modeling to solicit and evaluate multiple solutions from the community.
+Aims 1--3 focus on tailoring and generating this experimental data, while Aim 4 focuses on fielding annual SAMPL challenges.
+Aims 1--3 bring together multiple laboratories and both theorists and experimentalists: graduate students from the Mobley and Chodera laboratories are paired with well-equipped experimental groups in industry to collect physical property data (Aim 1); leading experimentalists in supramolecular chemistry Gibb and Isaacs work with theorist Mobley to perform host-guest affinity measurements (Aim 2); new approaches to the automated development and investigation of model protein-ligand systems are developed in the Chodera lab (Aim 3).
```

Comment:
I've written all my materials/biosketches/etc. with the assumption that both of our groups would contribute students for Aim 1. I have 9 students that could participate, all have some degree of experimental experience, and we have a track record of this now, so I felt this would be a strength, since your students do not necessarily have experimental experience.

Only Mehtap Isik (TPCB student working on model systems) will be fully dedicated to Aim 3. She would likely not participate in the industry internships unless as a result, unless we had someone else take over the Aim 3 project temporarily.